### ISSUE TITLE
Re-add TTY option for auto-starting

### ISSUE THREAD

**Issue by Frewacom** [engstrand-config/home-service-dwl-guile#7]
It is nice to have other TTY's as escape hatches in case dwl-guile and Sheperd decides to not play nice.

**Comment by m4lin** [comment]
Yes i think is my case. Dwl starts ok at the end of guix home reconfigure homeconf.scm. but if I reboot I do have only login at tty. How I suppose to start dwl-guile? I tried herd start dwl-guile. Does not work.

Thanks

Marco

**Comment by Frewacom** [comment]
dwl-guile will auto start on the first tty you login to (any tty), assuming that you have the dwl-guile home service in your config. You could check the startup log in `~/.local/var/log/dwl-guile.log` to see if there are any errors.

**Comment by m4lin** [comment]
Than there is something wrong in my config. But I do not get what.

(use-modules
 (gnu home services guix)
 (gnu services)
 (guix channels)
 (gnu packages emacs)
 (gnu packages)
 (guix gexp)
 (guixrus home services emacs)
 (gnu packages admin)
 (dwl-guile home-service)
 (dwl-guile patches)
 (guix packages)
 (gnu packages terminals)
 (gnu packages lxde)
 )

(define %services
  (list 
        (service home-dwl-guile-service-type
			 ;; If you wish to configure the home service further, you can pass in
			 ;; a configuration to the service. All options listed below are optional.
            (home-dwl-guile-configuration
		
			 ;; If dwl-guile should auto-start on first login. Enabled by default. #f to deactivate
            (auto-start? #t)
			 ;; change foot with sakura
             ;; Create a custom configuration for change foot with sakura.
			(config
				(dwl-config (terminal `(,(file-append sakura "/bin/sakura"))))
		    )
			)
		)
	)
)
(home-environment
(services %services)
 (packages (list
           htop
           pcmanfm
		   sakura
           )
  )
)

**Comment by Frewacom** [comment]
Looks okay to me. Did you check the log in `~/.local/var/log/dwl-guile.log`? I would assume that dwl-guile fails during startup.

**Comment by m4lin** [comment]
apparently i do not have that file in ~/.local/var/log/, i do have only shepherd.log which states the service started, which is true, but only after guix home reconfigure. at any other reboot dwl do not start at login and no additional records are in /.local/var/log/.

there is any way to start dwl if not starting by itself?

EDIT:

i cannot start the service with herd start dwl-guile, error 
error: connect: /run/user/1001/shepherd/socket: No such file or directory

**Comment by Frewacom** [comment]
Strange, sounds like it is shepherd itself that does not start correctly/crashes. I assume `herd status` doesn't work either? If shepherd is running and dwl-guile home service is activated, running that command should show the dwl-guile service in the output. 

> there is any way to start dwl if not starting by itself?

You should be able to start dwl-guile manually without herd using `dwl-guile -c ~/.config/dwl-guile/config.scm`.

**Comment by m4lin** [comment]
same error if i try to use herd and dwl-guile command not found, i am a bit confused now, if i reconfigure it starts so how it is possible it not there.

~$ herd status
error: connect: /run/user/1001/shepherd/socket: No such file or directory
~$ dwl-guile -c ~/.config/dwl-guile/config.scm
-bash: dwl-guile: command not found
 ~$

**Comment by Frewacom** [comment]
Yeah, sounds like an issue with Guix or your installation. Not sure what the issue is, sorry.
